storage/cloud-client/README.rst

```diff
@@ -154,7 +154,7 @@ To run this sample:
     
```

Comment:
Ahhhhh, I totally missed that.

Is there a place I could put the markdownful list of setup instructions? Putting it in the description of the python file seemed to distort things.